src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -581,6 +581,21 @@
     AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
```

Comment:
I am wondering if using varargs here makes sense, I think it makes these different methods ambigous?
what's the difference between `client.prepareAnalyze("index", "text1");` and `client.prepareAnalyze("text1", "text2");` and `client.prepareAnalyze("text1");`?

Maybe we should got for new methods that accept an array instead?